.all-contributorsrc

```diff
@@ -2609,6 +2609,15 @@
       "contributions": [
         "infra"
       ]
+    },
+    {
+      "login": "vadimshvetsov",
+      "name": "Vadim Shvetsov",
+      "avatar_url": "https://avatars.githubusercontent.com/u/16336572?v=4",
+      "profile": "https://proling.ru",
```

Comment:
@vadimshvetsov - Do you have a personal site? Linking to a brand is something we're not doing :)

Reply:
Hey @MatanBobi removed that, but feel free to not reopen this)